Sources/SQLKit/Expressions/Syntax/SQLRaw.swift

```diff
@@ -1,16 +1,23 @@
 public struct SQLRaw: SQLExpression {
     public var sql: String
-    public var binds: [any Encodable]
+
+    @available(*, deprecated, message: "Binds set in an `SQLRaw` are ignored. Use `SQLBind`instead.")
+    public var binds: [any Encodable] = []
     
     @inlinable
-    public init(_ sql: String, _ binds: [any Encodable] = []) {
+    public init(_ sql: String) {
+        self.sql = sql
+    }
+
+    @available(*, deprecated, message: "Binds set in an `SQLRaw` are ignored. Use `SQLBind`instead.")
+    @inlinable
+    public init(_ sql: String, _ binds: [any Encodable]) {
         self.sql = sql
         self.binds = binds
     }
     
     @inlinable
     public func serialize(to serializer: inout SQLSerializer) {
         serializer.write(self.sql)
-        serializer.binds += self.binds
```

Comment:
Just as a note, this removal makes this a (pretty big) breaking change. I'm not sure there's anything to be done about it now, since the release was, I believe, quite a while ago. But just a note for the future

Reply:
It was a very much deliberate breakage. This particular "functionality" of `SQLRaw` was a misfeature accidentally left in the code during Tanner's work on Vapor 4; it should never have been present to begin with. I couldn't remove the `binds` property itself without causing source breakage, but I could at least discourage its usage.

Reply:
_That being said..._

If you are a user whose code was broken by this change, I'm not dead-set against undoing it, even this long afterwards; putting it back would be a purely additive change. The `binds` property will be gone anyway in Fluent 5, so it doesn't matter as much for Fluent 4 at this point.

Reply:
I don't personally need it (I only used it in two places, and I figured out the `\(bind: bindVal)` thing). I just think it's _unexpected_ for a minor version bump to remove significant (though apparently unintentional) runtime functionality with only a compile-time warning as an indication

Idk if this project uses Semver. If not, then the expectation that that wouldn't happen is my fault, and you should just ignore me :)

Reply:
Just to nitpick 😅  - SemVer definitely allows breaking behaviour changes, otherwise you'd never be able to fix bugs as that's a change in behaviour. SemVer is purely about API changes, and in Swift's case, whether something still compiles, not if it behaves the same way (not saying that's any comfort for having something major changed and we tend to try and avoid that in most places)